client/components/mma/upgrade/UpgradeSupportStyles.ts

```diff
@@ -0,0 +1,48 @@
+import { css } from '@emotion/react';
+import { brand, headline, space, textSans } from '@guardian/source-foundations';
+
+export const sectionSpacing = css`
+	margin-top: ${space[4]}px;
+`;
+
+export const linkCss = css`
+	${textSans.medium()};
+	color: ${brand[400]};
+	font-weight: 700;
+	text-decoration-line: underline;
+	margin-top: 10px;
+	margin-left: 20px;
+	display: flex;
+	justify-content: center;
+`;
+
+export const headingCSS = css`
+	${headline.medium({ fontWeight: 'bold' })};
+	margin-top: 0;
+	margin-bottom: 0;
+`;
+
+export const paragraphCss = css`
+	margin-bottom: 32px;
```

Comment:
I struggled with this a bit because there is no space[8] in source, but I think we should try only use space units https://guardian.github.io/csnx/?path=/docs/source-foundations_space--docs instead of hard coding these pixel amounts

Reply:
what I did to get 32px was combine margin and padding

Reply:
What did you do to get the combined 32px?

Reply:
here's an example where I did that 
[link](https://github.com/guardian/manage-frontend/blob/3087460ef2b5f7e60d093a00ca7c2a9812a9584f/client/components/mma/upgrade/UpgradeSupport.tsx#L74)
because they're in a stack there is a margin, and then the individual component has some padding to it, this adds up to the space in the figma